### PR TITLE
Misc. fixes from PR 1504, without draft articles for tools

### PR DESCRIPTION
**Pull request by mikehoffms**
Motivation for creating this "1504 misc fixes" branch & PR, off of PR 1504 (branch: "feature-based-toc"):
1504 will probably be Closed, not Merged, because it won't be ready for a long time, because 13 articles are or were needed to be added there.  

This PR is needed, to carry forward not the individual missing tool articles like those which codepo8 added in 1504 (Changes tool) or captainbrosset (Search tool, Performance monitor tool); I created this branch & PR to carry forward misc fixes that were made in 1504, including mainly the following valuable changes:
* New article https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/about-panel-drawer-tools?branch=pr-en-us-1548
* Style guide wording updates to https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/accessibility-testing-in-devtools?branch=pr-en-us-1548
* New article broken out from long article for consistency: https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/narrow?branch=pr-en-us-1548
* cross-links for keyboard shortcuts
* Broke up table into sections: https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/reference?branch=pr-en-us-1548
* Removed deprecated :::row/col::: tagging from https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/api?branch=pr-en-us-1548
* Style guide rewording updates of https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/console-javascript?branch=pr-en-us-1548
* ...

### CODE REVIEW

**Review by vchapel — COMMENTED** [review]
All the x and + are code, which seems to be a longstanding format. But shouldn't they be bold like any other IU interaction item?

File: microsoft-edge/devtools-guide-chromium/about-panel-drawer-tools.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+<!-- ====================================================================== -->
+## Opening in tool in its default toolbar
+
+1.  Open the **Command Menu**.
+
+1.  Type the name of the tool, and then select the tool.
+
+    * If the tool has "Panel" next to it in the Command Menu, the tool opens in the main toolbar at the top of DevTools.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/about-panel-drawer-tools.md:63]:
```suggestion
    * If the tool has **Panel** next to it in the Command Menu, the tool opens in the main toolbar at the top of DevTools.
```

File: microsoft-edge/devtools-guide-chromium/about-panel-drawer-tools.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    * If the tool has "Drawer" next to it in the Command Menu, the tool opens in the Drawer toolbar.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/about-panel-drawer-tools.md:64]:
```suggestion
    * If the tool has **Drawer** next to it in the Command Menu, the tool opens in the Drawer toolbar.
```

**Review by vchapel — COMMENTED** [review]
Because of the age of this, I think the links are the old style [text][link name] instead of [text](actual link). It might be too time consuming and outside scope to be fixed in the PR.

File: microsoft-edge/devtools-guide-chromium/open/index.md [hunk 3]
```diff
@@ -47,31 +47,27 @@ For more information, navigate to [Microsoft Edge DevTools keyboard shortcuts][D
...
 
 <!--Navigate to [Get Started With The Console][ConsoleGetStarted].  -->
 
 
 <!-- ====================================================================== -->
 ## Open the previous panel
 
-To jump to the previously open panel, select `Control`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  For more information, navigate to [Microsoft Edge DevTools keyboard shortcuts][DevtoolsShortcutsIndex].
-
+To jump to the previously open panel, select `Control`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  See [Keyboard shortcuts][DevtoolsShortcutsIndex].
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/open/index.md:70]:
```suggestion
To jump to the previously open panel, press `Control`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  See [Keyboard shortcuts][DevtoolsShortcutsIndex].
```

**Review by vchapel — APPROVED** [review]
You deserve a Pulitzer for removing "canonical documentation."  Good progress overall.

**Comment by mikehoffms** [comment]
> All the x and + are code, which seems to be a longstanding format. But shouldn't they be bold like any other IU interaction item?

@vchapel - What's the context of this comment?

**Comment by mikehoffms** [comment]
> Because of the age of this, I think the links are the old style [text][link name] instead of [text](actual link). It might be too time consuming and outside scope to be fixed in the PR.

@vchapel - What's the context of this comment?

**Comment by vchapel** [comment]
> > Because of the age of this, I think the links are the old style [text][link name] instead of [text](actual link). It might be too time consuming and outside scope to be fixed in the PR.
> 
> @vchapel - What's the context of this comment?

Some files have the new simpler [text](direct link) and some files have [text][link from bottom of file] ... just noting it. Feel free to delete or resolve this comment as you like.

**Pull request merged**